### DISCUSSION TITLE
Can a Subfolder's Contents be Compared with Files Only?

### DISCUSSION THREAD

**Discussion by tatihulot** [WinMerge/winmerge#2373] [Q&A]
Hello, I’ve just begun experimenting with WinMerge, and it’s excellent.

However, there is something I’d like to do with it, and I haven’t figured out how yet.  I was hoping I could request some assistance.

I want to compare two folders.  The left folder contains only files.  The right folder contains subfolders.  Some of the subfolders contain files that are also on the left folder.

Is there a way to configure WinMerge to check the contents of the subfolders on the right side, and compare them to the files on the left?

For example, let’s say the Left Folder has these files in it:

**Mariebirthday.avi
Glennbirthday.avi
Annegraduation.avi**

On the right side, we have these subfolders, which contain the above files:

**Marie’s Birthday** (which contains mariebirthday.avi)
**Glenn’s Birthday** (which contains glennbirthday.avi)
**Anne’s High School Graduation** (which contains annegraduation.avi)

Can this be done with WinMerge?  Can the contents of subfolders be examined and compared?  If it can be done, what is the procedure?

Thank you, everyone.   Jd

**Comment by sdottaka** [comment]
Unfortunately, WinMerge does not have the ability to compare files ignoring the folder structure. Beyond Compare and other programs seem to have this functionality.

**Reply by tatihulot** [reply]
Thank you for the response.
